modules/installation-supported-aws-machine-types.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+// * installing/installing_aws/installing-aws-government-region.adoc
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws/installing-aws-vpc.adoc
+// * installing/installing_aws/installing-aws-private.adoc
+
+[id="installation-supported-aws-machine-types_{context}"]
+= Supported AWS machine types
+
+The following Amazon Web Services (AWS) instance types are supported with {product-title}.
+
+.Instance types for machines
```

Comment:
@kalexand-rh same as above two comments, please refer to https://github.com/openshift/installer/pull/4327/files

thanks